src/ac/templates/zds_schema/error_detail.html

```diff
@@ -0,0 +1,23 @@
+{% extends "master.html" %}
```

Comment:
The HTML files/css got changed recently to use Bootstrap. See ZRC for example.

Reply:
Yep, need to update the boilerplate project: https://github.com/VNG-Realisatie/gemma-api-boilerplate

Reply:
`zds_schema` als folder is sowieso niet meer gebruikt door de `vng-api-common` rename